frontend/public/components/chargeback.tsx

```diff
@@ -317,23 +233,28 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
 
   render() {
     const {obj} = this.props;
-    const phase = _.get(obj, ['status', 'phase']);
-
-    const {data, reduceBy, sortBy, orderBy, keys, rows, maxValues, totals, inFlight, error} = this.state;
+    const conditions = _.get(obj, 'status.conditions');
+    const isFinished = _.some(conditions, {reason: 'Finished'});
+    const errorCondition = _.find(conditions, (condition) => {
+      return (condition.type === 'Failure' && condition.status === 'True') ||
+        (condition.type === 'Running' && condition.status === 'False' && condition.reason !== 'Finished');
```

Comment:
Is this second check really needed? It seems like `Failure` / `True` should be enough. Is there a case where that doesn't get set?

Reply:
Yes, here are some errors, but Type === 'Running' && Status === 'False':
![image](https://user-images.githubusercontent.com/12733153/59205747-80347e80-8b71-11e9-9c29-44cf85ef478e.png)
![image](https://user-images.githubusercontent.com/12733153/59205805-a1956a80-8b71-11e9-8fe3-b9816c8a25ee.png)

I believe I only saw a Type === 'Failure' once when I didn't have the Metering installed correctly.

Reply:
Instead of checking all of these conditions, can we simply check if there is report data present? We show the conditions, so the user can see the problem.

Reply:
Fixed